arm/Microsoft.Compute/galleries/readme.md

```diff
@@ -1,13 +1,15 @@
-# Shared Image Gallery `[Microsoft.Compute/galleries]`
+# ComputeGalleries `[Microsoft.Compute/galleries]`
```

Comment:
This name is generated and a `best effort` kind of thing by the script. I'd suggest to rename it back to the orginal `Shared image gallery`. That's also how it will show up in the module tables